process/slash/detector/benchmarkMultipleHeaderProposal_test.go

```diff
@@ -0,0 +1,87 @@
+package detector_test
+
+import (
+	"testing"
+
+	coreSlash "github.com/ElrondNetwork/elrond-go-core/data/slash"
+	"github.com/ElrondNetwork/elrond-go-core/hashing"
+	"github.com/ElrondNetwork/elrond-go-core/hashing/blake2b"
+	crypto "github.com/ElrondNetwork/elrond-go-crypto"
+	"github.com/ElrondNetwork/elrond-go-crypto/signing"
+	"github.com/ElrondNetwork/elrond-go-crypto/signing/mcl"
+	"github.com/ElrondNetwork/elrond-go/process"
+	"github.com/ElrondNetwork/elrond-go/process/slash/detector"
+	"github.com/stretchr/testify/require"
+)
+
+func BenchmarkMultipleHeaderProposalDetector_VerifyData(b *testing.B) {
+	if testing.Short() {
```

Comment:
I don't think that the testing.Short is necessary. The Benchmarks should be run only with the -bench flag

Reply:
Right, removed them